src/serial.rs

```diff
@@ -100,6 +108,130 @@ cfg_if! {
     }
 }
 
+/// Types for configuring a serial interface.
+pub mod config {
+    // TODO: To export or not to export. As baud is used by this module,
+    // shouldn't this be automagically available to users of this module?
```

Comment:
The extension traits should already be pulled in via [`stm32f3xx-hal::prelude::*`](https://github.com/stm32-rs/stm32f3xx-hal/blob/66c0d21ae19ae0bee09ec834a6c9c90b2191e17d/src/prelude.rs#L18-L19)

Reply:
Done.